lib/l10n/intl_ro.arb

```diff
@@ -168,6 +168,7 @@
   "navigationEventDetails": "Detalii eveniment",
   "navigationNewsFeed": "Știri",
   "navigationClassInfo": "Informații materie",
+  "navigationSearch": "Caută",
```

Comment:
Căutare? As far as I can see this isn't an action (i.e. text on a button), but rather a page title, so it should be a noun. If you do wanna use it for a button, make a separate string ("actionSearch"?).